eng/Versions.props

```diff
@@ -73,6 +73,7 @@
   <PropertyGroup>
     <!-- Dependencies from https://github.com/mono/linker -->
     <MicrosoftNETILLinkTasksPackageVersion>6.0.0-alpha.1.21057.1</MicrosoftNETILLinkTasksPackageVersion>
+    <MicrosoftNETILLinkAnalyzerPackageVersion>$(MicrosoftNETILLinkTasksPackageVersion)</MicrosoftNETILLinkAnalyzerPackageVersion>
```

Comment:
I think it is - since the MicrosoftNETILLinkTasksPackageVersion is already flowing. I find it a bit odd to tie the versions together, but this seems to be the pattern used elsewhere in sdk.